bot/cogs/help_channels.py

```diff
@@ -457,6 +464,13 @@ def is_dormant_message(message: t.Optional[discord.Message]) -> bool:
             topic=DORMANT_TOPIC,
         )
 
+        self.bot.stats.incr(f"help.dormant_calls.{caller}")
+
+        if self.claim_times.get(channel.id):
+            claimed = self.claim_times[channel.id]
+            in_use_time = datetime.now() - claimed
+            self.bot.stats.timer("help.in_use_time", in_use_time)
```

Comment:
Need to suppress key errors since on bot restart claim times will not persist.

Reply:
That's what the if will do though right? If there is no entry in `claim_times` just move on.

Reply:
Right, sorry. Didn't notice the `get()`.

Reply:
Though an `in` would make more sense.